### ISSUE TITLE
Remove references to the input types still not well-supported in current browsers?

### ISSUE THREAD

**Issue by binlabs** [twbs/bootstrap#16181]
The [Inputs section](http://getbootstrap.com/css/#inputs) on the CSS page of the documentation really makes it sound like the new input types in HTML5, like the color input type, are well-supported.

> Most common form control, text-based input fields. Includes support for all HTML5 types: text, password, datetime, datetime-local, date, month, time, week, number, email, url, search, tel, and color.

Searching through the issues, the only discussions I can find relating to the newer input types like color are specifically in reference to the customizer, not the core Bootstrap scripts and styles.

I know that the reason there aren't any specific styles for these newer input types is that there just isn't wide enough browser support to do so yet, I just found it strange that the documentation seems to indicate these elements have some sort of support.

![Color Input Type using Chrome 41.0.2272.101 m on Windows 7](https://cloud.githubusercontent.com/assets/1139848/6894547/708475a0-d69c-11e4-9f37-2cd7ebeb1bf2.png)

Rather than just removing the second sentence in that paragraph from [docs/_includes/css/forms.html](https://github.com/twbs/bootstrap/blob/master/docs/_includes/css/forms.html) and submitting a pull request, I thought I'd raise the issue here first to see if I was reading that part wrong.

Basically, there is no special support whatsoever for the HTML5 input types like color, so I suggest we remove the sentence that suggests that there is support from the documentation, or add some sort of disclaimer.

Here is a [fiddle](https://jsfiddle.net/binvisions/ksumjx3a/) so that you can see how the color input type looks for you.

**Comment by patrickhlauke** [comment]
Depends how you define "support" I guess. With the exception of the problem with the color picker one you noted, all other input types receive the correct basic rounded corners, glowing focus indication, etc. and can be used with additions like the optional icons, validation states, etc.

**Comment by mdo** [comment]
Yeah, that section of the docs is meant to say you can use the `.form-control` class on any textual input, including the ones listed. It's not to say we provide special support for all those controls' interactions, just their initial styling.
